src/main/java/org/broadinstitute/consent/http/resources/DatasetResource.java

```diff
@@ -660,15 +659,9 @@ public Response searchDatasets(
   public Response searchDatasetIndex(@Auth AuthUser authUser, String query) {
```

Comment:
I think we need a `@PermitAll` line for this method - I definitely missed this in my earlier review.